Tests/test_content.py

```diff
@@ -306,10 +305,13 @@ def load_conf_files(conf_path, secret_conf_path):
 
 
 def load_env_results_json():
-    if not os.path.isfile(ENV_RESULTS_PATH):
+    env_results_path = os.getenv('ENV_RESULTS_PATH', os.path.join(os.getenv('ARTIFACTS_FOLDER', './artifacts'),
```

Comment:
Why have you changed it so that the variable is local to this function? You repeat it down below.